internal/cmd/base/initial_resources.go

```diff
@@ -125,78 +125,110 @@ func (b *Server) CreateInitialAuthMethod(ctx context.Context) (*password.AuthMet
 	b.InfoKeys = append(b.InfoKeys, "generated auth method id")
 	b.Info["generated auth method id"] = b.DevAuthMethodId
 
-	// Create the dev user
-	if b.DevLoginName == "" {
-		b.DevLoginName, err = base62.Random(10)
+	createUser := func(loginName, loginPassword, userId string, admin bool) (*iam.User, error) {
```

Comment:
that's a really long closure... easy to read though.

Reply:
I agree, it's long, but it does a related chunk of work...I found myself duplicating all of that, which meant that the unit of work was...all of that. :-)